.github/workflows/metrics.yml

```
@@ -0,0 +1,46 @@
+name: metrics
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: startsWith(github.repository, 'path-cc/')
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: setup and run metrics
+        run: |
+          cd campuses-with-activate-researchers
+          python3 -m venv .
+          . ./bin/activate
+          pip install -r requirements.txt
+          ./campuses-with-active-researchers.py --csv > ../campuses-with-active-researchers.csv
+          cd ..
+          ./campus-contributions --json > ../campus-contributions.json
+
+    - name: Commit files
+      run: |
+        git clone --depth=1 git@github.com:path-cc/metrics
+        mv campus-contributions.json metrics
+        mv campuses-with-active-researchers.csv metrics
+        cd metrics
+        git config --local user.email "help@opensciencegrid.org"
+        git config --local user.name "Automatic preview publish"
+        git add campuses-with-active-researchers.csv
+        git add campus-contributions.json
+        git commit -m "nightly metrics update"
```

```suggestion
        git commit -m "nightly metrics update (from ${GITHUB_REPOSITORY} ${GITHUB_WORKFLOW} ${GITHUB_RUN_ID}.${GITHUB_RUN_NUMBER})"
```

For more details using https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables